src/content.py

```diff
@@ -136,4 +136,13 @@ def check_for_incomplete_quarterly_data(df, new_index, quarterly):
     if pd.isna(quarterly[column_name].iloc[-1]):
         wmfdata.utils.print_err("This quarterly report is based on incomplete data as some months' data is not available.")
     
-    return quarterly_averages
\ No newline at end of file
+    return quarterly_averages
+
+
+# Saves quarterly metrics in a tsv file.
+def generate_quarterly_metric(quarterly_averages, core_metrics, index_names, quarterly_averages_content_gap):
```

Comment:
This is a lot of customizability that we are never going to use (if anything, we'd be more likely to change the file name, which isn't customizable). In my opinion, it makes the code harder to follow for no good reason. 

In the short term, I'd take an approach similar to `process_quality_data` or `calculate_mom`, in which pretty much everything is hard-coded.

In the long term, if we really want to build this to make it easy to change the selection of quarterly metrics (which might be a good idea), we should use a class rather than relying on the caller to pass the arguments back and forth between different functions.